_includes/rems-link.md

```diff
@@ -0,0 +1 @@
+<a class="button" href="https://rems.dsp.aida.scilifelab.se/apply-for?resource={{ page.datacite.["@id"] }}">Apply for access</a>"
```

Comment:
I'm not sure exactly how to design it, but I think we should somehow offer help close by here (link to https://docs.datahub.aida.scilifelab.se/data/data-sharing-request/)

Reply:
hmm, something like "Refer to the guide for dataset application"?